main.js

```diff
@@ -16,18 +16,22 @@ const removeClass = (el, oldClass) => {
   const inputField = document.getElementsByTagName('input')[0];
   let typeTimeout;
   const TYPE_INTERVAL = 10000;
+  // Set variable for filtering purposes
   const siteContainer = document.getElementsByTagName('body');
```

Comment:
Just noticed: could the array index [0] go at the end of this variable value, rather than on line 62? @heathervv